projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -367,6 +373,19 @@ private function is_allowed_plugin_handle( $handle ) {
 		return false;
 	}
 
+	/**
+	 * Convert relative URLs to absolute URLs.
+	 *
+	 * @param string $src The source URL.
+	 * @return string The absolute URL.
+	 */
+	public function make_url_absolute( $src ) {
+		if ( ! empty( $src ) && strpos( $src, '/' ) === 0 && strpos( $src, '//' ) !== 0 ) {
```

Comment:
(nit) `strpost` could be optimized: https://stackoverflow.com/a/2790919

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/45358.

Reply:
Thanks for following up!